installing/installing_aws/aws-compute-edge-zone-tasks.adoc

```diff
@@ -62,20 +62,20 @@ include::modules/machineset-creating.adoc[leveloffset=+3]
 [role="_additional-resources"]
 .Additional resources
 
-* xref:../installing/installing_aws/ipi/installing-aws-localzone.adoc#installing-aws-localzone[Installing a cluster on AWS with compute nodes on AWS Local Zones]
-* xref:../installing/installing_aws/ipi/installing-aws-wavelength-zone.adoc#installing-aws-wavelength-zone[Installing a cluster on AWS with compute nodes on AWS Wavelength Zones]
+* xref:../../installing/installing_aws/ipi/installing-aws-localzone.adoc#installing-aws-localzone[Installing a cluster on AWS with compute nodes on AWS Local Zones]
+* xref:../../installing/installing_aws/ipi/installing-aws-wavelength-zone.adoc#installing-aws-wavelength-zone[Installing a cluster on AWS with compute nodes on AWS Wavelength Zones]
 
 // Creating user workloads in AWS Local Zones or Wavelength Zones
 include::modules/installation-extend-edge-nodes-aws-local-zones.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
 
-* xref:../installing/installing_aws/ipi/installing-aws-localzone.adoc#installing-aws-localzone[Installing a cluster on AWS with compute nodes on AWS Local Zones]
-* xref:../installing/installing_aws/ipi/installing-aws-wavelength-zone.adoc#installing-aws-wavelength-zone[Installing a cluster on AWS with compute nodes on AWS Wavelength Zones]
-* xref:../nodes/scheduling/nodes-scheduler-taints-tolerations.adoc#nodes-scheduler-taints-tolerations-about_nodes-scheduler-taints-tolerations[Understanding taints and tolerations]
+* xref:../../installing/installing_aws/ipi/installing-aws-localzone.adoc#installing-aws-localzone[Installing a cluster on AWS with compute nodes on AWS Local Zones]
+* xref:../../installing/installing_aws/ipi/installing-aws-wavelength-zone.adoc#installing-aws-wavelength-zone[Installing a cluster on AWS with compute nodes on AWS Wavelength Zones]
+* xref:../../nodes/scheduling/nodes-scheduler-taints-tolerations.adoc#nodes-scheduler-taints-tolerations-about_nodes-scheduler-taints-tolerations[Understanding taints and tolerations]
 
 [id="next-steps_aws-zone-tasks"]
 == Next steps
 
-* Optional: Use the AWS Load Balancer (ALB) Operator to expose a pod from a targeted edge compute node to services that run inside of a {zone-type} subnet from a public network. See xref:../networking/aws_load_balancer_operator/install-aws-load-balancer-operator.adoc#nw-installing-aws-load-balancer-operator_aws-load-balancer-operator[Installing the AWS Load Balancer Operator].
+* Optional: Use the AWS Load Balancer (ALB) Operator to expose a pod from a targeted edge compute node to services that run inside of a {zone-type} subnet from a public network. See xref:../../networking/aws_load_balancer_operator/install-aws-load-balancer-operator.adoc#nw-installing-aws-load-balancer-operator_aws-load-balancer-operator[Installing the AWS Load Balancer Operator].
```

Comment:
```suggestion
* Optional: Use the AWS Load Balancer (ALB) Operator to expose a pod from a targeted edge compute node to services that run inside of a {zone-type} subnet from a public network. See xref:../../networking/aws_load_balancer_operator/install-aws-load-balancer-operator.adoc#nw-installing-aws-load-balancer-operator_aws-load-balancer-operator[Installing the AWS Load Balancer Operator by using the web console].
```

Reply:
Updated this to just the general link to the page rather than the specific web console procedure